src/state/shellscripts/bwrap.sh

```diff
@@ -11,6 +11,13 @@ if ! command -v bwrap >/dev/null; then
     exit 10
 fi
 
+OPAMVERSION=`opam --version`
```

Comment:
_(putting back @AltGr that I somehow deleted when deleting my original comment that opened this block)_

uuh... calling opam from here ??
Also we have no guarantee that the calling process is actually the same version as opam looked up in the PATH here ?